src/code.cloudfoundry.org/gorouter/route/pool.go

```diff
@@ -74,6 +74,21 @@ type ProxyRoundTripper interface {
 	CancelRequest(*http.Request)
 }
 
+type HashRoutingProperties struct {
+	Header        string
+	BalanceFactor float64
```

Comment:
Can you please add a comment explaining what this value means and what the range of valid values is?